src/lib/db/query.js

```diff
@@ -10,9 +11,9 @@ const groupByType = ' GROUP BY dataset';
 const groupByDateTypeUser = ' GROUP BY date_time::date, dataset, username ORDER BY date_time::date';
 const groupByTypeGroup = 'GROUP BY name, dataset';
 const totalCount = 'SELECT count(*)::INTEGER FROM lev_audit';
-const forToday = ' WHERE date_time::DATE = current_date';
-const fromDate = 'date_time::DATE >= $(from)';
-const toDate = 'date_time::DATE < $(to)';
+const forToday = ` WHERE (CURRENT_TIMESTAMP ${timeZone})::DATE = (date_time ${timeZone})::DATE;`;
```

Comment:
It looks like we have a lot of `::DATE` truncation of `date_time`, i.e. everywhere the column is used, so the following might be useful:
```js
const DTZ = `(date_time ${timeZone})::DATE`;
```

Reply:
Refactored